Source/artifacts/package.json

```diff
@@ -37,8 +37,7 @@
         "@dolittle/rudiments": "5.0.1",
         "@dolittle/types": "5.0.1",
         "@dolittle/runtime.contracts": "5.1.19",
-        "is-natural-number": "^4.0.1",
-        "rxjs": "6.6.0"
+        "is-natural-number": "^4.0.1"
```

Comment:
In the spirit of limiting dependencies to reduce our exposure, can we get rid of this package and just implement the logic ourselves?

It is really just `Number.isSafeInteger(value) && value > 0`.

Reply:
Done